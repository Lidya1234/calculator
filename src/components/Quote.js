import * as styles from '../style.module.css';

const Quote = () => (
  <div className={styles.quote}>
    <h1>Quote Of The Day</h1>
    <p>

      Mathematics is not about numbers, equations,
      computations, or algorithms: it is about understanding.
      â€“William Paul ThurstonHomeCalculatorQuote

    </p>
  </div>
);

export default Quote;
