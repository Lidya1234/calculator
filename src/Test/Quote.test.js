import React from 'react';
import { render } from '@testing-library/react';

import Quote from '../components/Quote';
import '@testing-library/jest-dom/extend-expect';

describe('Home has ', () => {
  it('Title of math Quotes', () => {
    const { getByText } = render(<Quote />);
    const title = getByText('Quote Of The Day');
    expect(title).toBeInTheDocument();
  });
  it('same content with the snapshot', () => {
    const { getByText } = render(<Quote />);
    const title = getByText('Quote Of The Day');
    expect(title).toMatchSnapshot();
  });
});
