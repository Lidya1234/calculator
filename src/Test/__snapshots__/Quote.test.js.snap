// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home has  same content with the snapshot 1`] = `
<h1>
  Quote Of The Day
</h1>
`;
